tf2_eigen/include/tf2_eigen/tf2_eigen.hpp

```diff
@@ -131,7 +131,7 @@ void doTransform(
 /** \brief Apply a geometry_msgs TransformStamped to a 6-long Eigen vector.
  * Useful for transforming wrenches or twists.
  * Wrench: (force-x, force-y, force-z, torque-x, torque-y, torque-z)
- * Twist: (trans. vel. x, trans. vel. y, trans. vel. z, ang. vel. x, ang. vel. y, ang. vel. z)
+ * Twist: (ang. vel. x, ang. vel. y, ang. vel. z, trans. vel. x, trans. vel. y, trans. vel. z)
```

Comment:
If you look at Eq. 3.83 here, this was wrong.

http://hades.mech.northwestern.edu/images/7/7f/MR.pdf